### DISCUSSION TITLE
Error in StaMPS processing: step 7; subr_tropo

### DISCUSSION THREAD

**Discussion by zrcunique** [alexisInSAR/EZ-InSAR#51] [Help]
Hello Dr. alexis,
I tried to reduce atmospheric InSAR noise with StaMPS processing but failed. Specifically, I set the parameter of subr_tropo to 'y', and the program reported an error in step 7, both in PS approach and SBAS approach. 

Moreover, besides the default method, I wonder if EZ-InSAR offers the option to download GACOS and apply it to reduce atmospheric effects?

Sincerely,
Zhou

<img width="388" alt="PS-error" src="https://github.com/alexisInSAR/EZ-InSAR/assets/134569511/04fb77c3-794c-40b2-aef9-6d5963bbc36e">
<img width="394" alt="SBAS-error" src="https://github.com/alexisInSAR/EZ-InSAR/assets/134569511/ec3e95a4-ee74-4b6c-9600-a2e7ad6c30bd">

**Comment by alexisInSAR** [comment]
Hi, 

Did you install TRAIN? You need to initialise the TRAIN processing directly in Matlab. 

For this moment, no but we work on its implementation. 

Kind regards, 

Alexis

**Reply by zrcunique** [reply]
I have checked the installation of the TRAIN and it did. It is not clear if TRAIN is called in StaMPS processing. Could you tell me the way to initialise TRAIN in Matlab? Thank you a lot!
